hera_qm/metrics_io.py

```diff
@@ -564,14 +595,21 @@ def _parse_dict_of_dict_of_dicts(input_str, value_type=float):
         output: dictionary of dictionary of dictionaries
     """
     # use regex to extract dictionaries
+    list_regex = r"\[.*\]"
+    list_match = re.match(list_regex, input_str)
+
+    if list_match:
+        return _parse_list_of_dict_of_dicts(input_str, value_type=value_type)
+
     dict_regex = r"([0-9]*?)\'?: \{(.*?\})\}"
     dicts = re.findall(dict_regex, input_str)
 
     # initialize output
     output = {}
     for d in dicts:
         # key is first capture group
-        key = int(str(d[0]))
+        # key = int(str(d[0]))
```

Comment:
This should be removed instead of commented out.

Reply:
oh good catch thanks. I commented it to test using the internal parser and never removed